source/sass/components/_footer.scss

```diff
@@ -70,6 +70,46 @@
     background-color: $color--white;
   }
 
+  &__language-switcher {
+    width: 100%;
+    margin-bottom: 32px;
+    display: flex;
+    -webkit-box-orient: horizontal;
```

Comment:
As something that's got somewhat [wonky support](https://caniuse.com/#search=box-orient) we probably want to keep this simpler and just rely on flex?

Reply:
Right, this was not needed, only kept `flex-direction: row;`